tutorials/speaker_tasks/ASR_with_SpeakerDiarization.ipynb

```diff
@@ -485,42 +447,31 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "color=get_color(signal,labels)\n",
-    "show_figure(signal,'audio with speaker labels',color)\n",
-    "display(Audio(signal,rate=16000))"
+    "transcription_path_to_file = f\"{ROOT}/asr_with_diartranscript_with_speaker_labels/an4_diarize_test.txt\"\n",
+    "transcript = read_file(transcription_path_to_file)\n",
+    "\n",
+    "pp.pprint(transcript)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Finally transcribe audio with time stamps and speaker label information"
+    "Another output is transcription output in JSON format, which is saved in `./asr_with_diar/json_result`. In the JSON format output, we include information such as transcription, estimated number of speakers (variable named `speaker_count`), start and end time of each word and most importantly, speaker label for each word."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": true
+   },
    "outputs": [],
    "source": [
-    "pos_prev = 0\n",
-    "idx=0\n",
-    "start_point,end_point,speaker=labels[idx].split()\n",
-    "print(\"{} [{:.2f} - {:.2f} sec]\".format(speaker,float(start_point),float(end_point)),end=\" \")\n",
-    "for j, spot in enumerate(spaces):\n",
-    "    pos_end = offset + (spot[0]+spot[1])/2*time_stride\n",
-    "    if pos_prev < float(end_point):\n",
-    "        print(words[j],end=\" \")\n",
-    "    else:\n",
-    "        print()\n",
-    "        idx+=1\n",
-    "        start_point,end_point,speaker=labels[idx].split()\n",
-    "        print(\"{} [{:.2f} - {:.2f} sec]\".format(speaker,float(start_point),float(end_point)),end=\" \")\n",
-    "        print(words[j],end=\" \")\n",
-    "    pos_prev = pos_end\n",
-    "\n",
-    "\n",
-    "print(words[j+1],end=\" \")"
+    "transcription_path_to_file = f\"{ROOT}/json_result/an4_diarize_test.json\"\n",
```

Comment:
path should be `{ROOT}/asr_with_diar/json_result/an4_diarize_test.json\`

Reply:
fixed